.travis.yml

```diff
@@ -42,9 +42,10 @@ install:
     - pip install codecov
 
 script:
-    - cd $TRAVIS_BUILD_DIR/dist/Orange-*
+    - cd "$ORANGE_DIR"
     - python -c "from Orange.tests import *"
-    - PYTHONUNBUFFERED=x coverage run --source=Orange setup.py test
+    - cp "$TRAVIS_BUILD_DIR"/.coveragerc ./  # for covereage and codecov
```

Comment:
Was this config intended to be a part of this pull request?

If so, it could be just put into the root folder.